scripts/run_jmeter_benchmark_for_jenkins_api.sh

```diff
@@ -1,12 +1,8 @@
-#!/bin/bash
+#!/bin/sh
 
-# The PWD is the root of the project, so create the path to the jmeter test file
-jmeter_file_path="$PWD/gumby/experiments/tribler/api_benchmark/$TEST_RUNNER_FILE"
-
-# jmeter has no output directory option, so CD into this directory and run it in there.
-pushd $OUTPUT_DIR
+# jmeter has no output directory option, so CD into this directory and run it in there
+cd ${OUTPUT_DIR}
 
 # Run jmeter headless (-n option) and specify the file using -t
-jmeter -n -t $jmeter_file_path
+jmeter -n -t ${EXPERIMENT_DIR}
```

Comment:
```${EXPERIMENT_DIR}``` in this case ```api_benchmark``` which contains ```run_requests_benchmark.conf``` which defines ```test_runner_file = request_benchmark_tribler_api.jmx```
https://github.com/brussee/gumby/blob/c2937ba0698b014076641653c8f25d8c85c95804/experiments/tribler/api_benchmark/run_requests_benchmark.conf